pkg/cmd/pulumi/stack_output.go

```diff
@@ -160,6 +172,32 @@ func (*jsonStackOutputWriter) WriteMany(outputs map[string]interface{}) error {
 	return printJSON(outputs)
 }
 
+// shellStackOutputWriter prints stack outputs as a shell script.
+type shellStackOutputWriter struct {
+	W io.Writer
+}
+
+var _ stackOutputWriter = (*shellStackOutputWriter)(nil)
+
+func (w *shellStackOutputWriter) WriteOne(v interface{}) error {
+	_, err := fmt.Fprintln(w.W, stringifyOutput(v))
```

Comment:
I would have expected this to still emit something like %s=%s?

Reply:
I was having trouble deciding.
Yeah, I think key=value, even for the single case makes sense for the shell script.
If someone wants just the output, they can drop the `--shell` flag.